kin-sdk/kin-sdk-lib/src/androidTest/kotlin/kin/sdk/KinAccountIntegrationTest.kt

```diff
@@ -399,7 +399,7 @@ class KinAccountIntegrationTest {
     }
 
     companion object {
-
+        //TODO why not to do just @BeforeClass and inside do fakeKinOnBoard = FakeKinOnBoard(), why companion object?
```

Comment:
because `BeforeClass` must be static. in Java it looks like disturbing... actually `FakeKinOnBoard` probably can be instantiated before any tests.